### PR TITLE
Yazan/model api implementation

### PR DESCRIPTION
**Pull request by santosjem**
imported constants to make calling contants shorter, made code more readable by lining variables up. Used the correct indentation. fixed xml files

### CODE REVIEW

**Comment by LarryAsante** [comment]
@YazanKittaneh : I'll let you review this and merge it if it meets the standard since you're more familiar with the work @santosjem is doing.

**Comment by LarryAsante** [comment]
@YazanKittaneh @santosjem: Status? If the only thing holding this back is the testing, let's merge and move on. We'll handle test coverage later

File: app/src/main/res/values/strings.xml
Comment (YazanKittaneh) [inline app/src/main/res/values/strings.xml]: Accidental refactoring happened here. Supposed to stay 'title' but changed t mTitle.

**Review by YazanKittaneh — APPROVED** [review]
All looks great. Just one refactoring technical error. Not major at all, and can be fixed in another pull request easily. Great job Jemuel!

**Pull request closed without merging**
